frontend/src/pages/CollaborationReportForm/index.js

```diff
@@ -0,0 +1,504 @@
+import { useHistory, Redirect } from 'react-router-dom';
+import { Alert, Grid } from '@trussworks/react-uswds';
+import { REPORT_STATUSES, DECIMAL_BASE } from '@ttahub/common';
+import moment from 'moment';
+import useDeepCompareEffect from 'use-deep-compare-effect';
+import pages from './Pages';
+import ActivityReportNavigator from '../../components/Navigator/ActivityReportNavigator';
+import './index.scss';
+import { NOT_STARTED } from './constants';
+import {
+  LOCAL_STORAGE_CR_DATA_KEY,
+  LOCAL_STORAGE_CR_ADDITIONAL_DATA_KEY,
+  LOCAL_STORAGE_CR_EDITABLE_KEY,
+} from '../../Constants';
+import { getRegionWithReadWrite } from '../../permissions';
+import useTTAHUBLocalStorage from '../../hooks/useTTAHUBLocalStorage';
+import { getCollaborators } from '../../fetchers/collaborators';
+import useLocalStorage, { setConnectionActiveWithError } from '../../hooks/useLocalStorage';
+import NetworkContext, { isOnlineMode } from '../../NetworkContext';
+import UserContext from '../../UserContext';
+import MeshPresenceManager from '../../components/MeshPresenceManager';
+
+// Default values for a new collaboration report go here
+const defaultValues = {
+  collaborators: [],
+};
+
+const pagesByPos = keyBy(pages.filter((p) => !p.review), (page) => page.position);
+const defaultPageState = mapValues(pagesByPos, () => NOT_STARTED);
+
+export function cleanupLocalStorage(id, replacementKey) {
+  try {
+    if (replacementKey) {
+      window.localStorage.setItem(
+        LOCAL_STORAGE_CR_DATA_KEY(replacementKey),
+        window.localStorage.getItem(LOCAL_STORAGE_CR_DATA_KEY(id)),
+      );
+      window.localStorage.setItem(
+        LOCAL_STORAGE_CR_EDITABLE_KEY(replacementKey),
+        window.localStorage.getItem(LOCAL_STORAGE_CR_EDITABLE_KEY(id)),
+      );
+      window.localStorage.setItem(
+        LOCAL_STORAGE_CR_ADDITIONAL_DATA_KEY(replacementKey),
+        window.localStorage.getItem(LOCAL_STORAGE_CR_ADDITIONAL_DATA_KEY(id)),
+      );
+    }
+
+    window.localStorage.removeItem(LOCAL_STORAGE_CR_DATA_KEY(id));
+    window.localStorage.removeItem(LOCAL_STORAGE_CR_ADDITIONAL_DATA_KEY(id));
+    window.localStorage.removeItem(LOCAL_STORAGE_CR_EDITABLE_KEY(id));
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.warn('Local storage may not be available: ', e);
+  }
+}
+
+function CollaborationReport({ match, location, region }) {
+  const { params: { currentPage, collabReportId } } = match;
+
+  const history = useHistory();
+  const [error, updateError] = useState();
+  const [loading, updateLoading] = useState(true);
+
+  const [lastSaveTime, updateLastSaveTime] = useState(null);
+
+  const [presenceData, setPresenceData] = useState({
+    hasMultipleUsers: false,
+    otherUsers: [],
+    tabCount: 0,
+  });
+  const [shouldAutoSave, setShouldAutoSave] = useState(true);
+
+  const [formData, updateFormData, localStorageAvailable] = useTTAHUBLocalStorage(
+    LOCAL_STORAGE_CR_DATA_KEY(collabReportId), null,
+  );
+
```

Comment:
@Andrew565 I understand why you are including this but I would suggest using the training report form as a base and not the activity report. Notably, the pattern established here (maintaining form state with react and also with react hook form) has lots of issues and I would not recommend it. Feel free to ping me if you want me to explain further